lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -6,11 +6,11 @@ module DateHelper
       alias_method :_distance_of_time_in_words, :distance_of_time_in_words
       alias_method :_time_ago_in_words, :time_ago_in_words
 
-      include DOTIW::Methods
-
       def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options = {}, options = {})
         return _distance_of_time_in_words(from_time, to_time, options) if options.delete(:vague)
-        super
+        dotiw_methods = Object.new
+        dotiw_methods.extend(DOTIW::Methods)
+        dotiw_methods.distance_of_time_in_words(from_time, to_time, include_seconds_or_options, options)
```

Comment:
Not sure if this is the cleanest way to call instance methods in the `DOTIW::Methods` module. Open to a better implementation if one exists.

Reply:
Ideally I want to write `DOTIW::Methods.distance_of_time_in_words(...)`

Reply:
Yeah, I tried that with no luck. I reasoned it was because `DOTIW::Methods` contains methods defined without `self.`, so I couldn't call them directly like that.

Reply:
Achieved: https://github.com/radar/distance_of_time_in_words/pull/111/commits/0636e3d3740f2656a267a03c56f7ce2082363d11

Reply:
👍